src/runtime/builder.c

```diff
@@ -191,7 +191,7 @@ int flatcc_builder_default_alloc(void *alloc_context, iovec_t *b, size_t request
 #define table_limit (FLATBUFFERS_VOFFSET_MAX - field_size + 1)
 #define data_limit (FLATBUFFERS_UOFFSET_MAX - field_size + 1)
 
-#define set_identifier(id) memcpy(&B->identifier, (id) ? (void *)(id) : (void *)_pad, identifier_size)
+#define set_identifier(id) strncpy((char * restrict)&B->identifier, (id) ? (void *)(id) : (void *)_pad, identifier_size)
 
```

Comment:
`restrict` is not portable and breaks MSVC build.
Also, when using type hashes, identifiers can be binary, so strncpy will not do.

Reply:
The restrict keyword can be dropped easy enough, but fixing the copy with padding is different. I'll see about reverting this line and fixing the padding.